### DISCUSSION TITLE
Gallery Box Font Size

### DISCUSSION THREAD

**Discussion by tpcdaz** [lllyasviel/stable-diffusion-webui-forge#778] [Q&A]
I was wondering if anyone knew how to change the text / font size that appears inside the gallery box that shows the Checkpoint / Lora title etc as some names wrap around. See photo. Thank you

![Screenshot 2024-06-01 190504](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/110419729/ddaca210-495f-4649-b6ed-db587e07457b)

**Comment by abzaloff** [comment]
Settings => Extra Networks => Card text scale (1 = original size) => change the value => Apply settings => Reload UI
